index.bs

```diff
@@ -879,7 +883,8 @@ below and in [[#index-defined-elsewhere]].
 :: This specification describes the syntax of all [=CBOR=]-encoded data using the CBOR Data Definition Language (<dfn>CDDL</dfn>) [[!RFC8610]].
 
 : COSE
-:: CBOR Object Signing and Encryption (COSE) [[!RFC8152]].  The IANA COSE Algorithms registry [[!IANA-COSE-ALGS-REG]] established by this specification is also used.
+:: CBOR Object Signing and Encryption (COSE) [[!RFC9052]] [[!RFC9053]].
+    The IANA COSE Algorithms registry [[!IANA-COSE-ALGS-REG]] maintained by these specifications is also used.
```

Comment:
Alternatively, this could be:

```suggestion
:: CBOR Object Signing and Encryption (COSE) [[!RFC9052]] [[!RFC9053]].
    The IANA COSE Algorithms registry [[!IANA-COSE-ALGS-REG]] originally established by [[!RFC8152 obsolete]] and updated by these specifications is also used.
```

Anyone have an opinion favouring either option?

Reply:
The latter seems marginally more helpful, but no strong feelings either way.